### ISSUE TITLE
Syncing account on dav.openxchange.eu

### ISSUE THREAD

**Issue by niklaas** [pimutils/vdirsyncer#762]
Hello, did anyone ever sync an account on openxchange.eu successfully? I ran into the following issue:

```
error: The server returned something vdirsyncer doesn't understand. Error message: InvalidXMLResponse('Invalid XML encountered: mismatched tag: line 1, column 2777\nDouble-check the URLs in your config.',)
error: While this is most likely a serverside problem, the vdirsyncer devs are generally interested in such bugs. Please report it in the issue tracker at https://github.com/pimutils/vdirsyncer/issues
error: 1 out of 1 tasks failed.
```

I have similar problems with other Cal/CardDAV clients as well. Each of these not managing to find the avilable address books and/or calendars. The only working client I found is OX Sync [1] that offers a distinct selection of providers. This selection includes EuroDNS which is behind openxchange.eu.

OX Sync uses dmfs [2] which manages to connect to `https://dav.openxchange.eu` successfully. So I started to reverse engineer it a bit, trying to find out which URLs it attempts to connect. Unfortunately, I could not find out.

Before spending more time on this (taking into account my lack of knowledge about Cal/CardDAV): Has anyone ever synced an account on dav.openxchange.eu successfully?

[1]: https://play.google.com/store/apps/details?id=com.openexchange.mobile.syncapp.enterprise
[2]: https://github.com/dmfs/jdav

**Comment by WhyNotHugo** [comment]
There's some instructions [here](http://oxpedia.org/wiki/index.php?title=AppSuite:CalDAVClients#Discover_the_CalDAV_URL_of_your_Calendar_Folders) on how to obtain the exact URL of your calendar. Have you tried using that URL?

**Comment by niklaas** [comment]
Yes, Ive also came across that article and tried that URL without any success.

I will provide logs of vdirsyncer and the official sync app for Android on 
the weekend. In the logs of the app several endpoints are polled  
unsuccessfully too when finally one of them works. I guess that should hint 
toward the right direction for getting vdirsyncer work with this particular 
provider.

Am 5. August 2018 8:16:07 nachm. schrieb Hugo Osvaldo Barrera 
<notifications@github.com>:
